transports/jms/src/test/java/org/mule/transport/jms/vendors/ActiveMQJmsConnectorTestCase.java

```diff
@@ -105,4 +109,23 @@ public void testCustomActiveMqConnectorConfig() throws Exception
 
         assertEquals("1.1", c.getSpecification()); // 1.0.2b is the default, should be changed in the config
     }
+
+    /**
+     * See MULE-8221
+     */
+    @Test
+    public void testActiveMqConnectorWithUsernameAndPassword() throws Exception
+    {
+        JmsConnector c = (JmsConnector) muleContext.getRegistry().lookupConnector("activeMqJmsConnectorWithUsernameAndPassword");
+
+        assertTrue(c instanceof ActiveMQJmsConnector);
```

Comment:
won't this fail on WMQ? This tests are imported by it and this assertion will fail